tests/arcus/ArcusCommunicationPrivateTest.cpp

```diff
@@ -188,16 +199,10 @@ TEST_F(ArcusCommunicationPrivateTest, ReadMeshGroupMessage)
 
     // - - Add settings to the mesh:
     std::map<std::string, std::string> mesh_settings = {
-        {"extruder_nr", "0"},
-        {"center_object", "1"},
-        {"mesh_position_x", "0"},
-        {"mesh_position_y", "0"},
-        {"mesh_position_z", "0"},
-        {"infill_mesh", "0"},
-        {"cutting_mesh", "0"},
-        {"anti_overhang_mesh", "0"},
+        { "extruder_nr", "0" },     { "center_object", "1" }, { "mesh_position_x", "0" }, { "mesh_position_y", "0" },
+        { "mesh_position_z", "0" }, { "infill_mesh", "0" },   { "cutting_mesh", "0" },    { "anti_overhang_mesh", "0" },
```

Comment:
Why are these two lines aligned? They don't belong to each other as far as I can tell. It's just a list of settings.

Reply:
Will check it out